program/CORE-V IDE prelminary project proposal.md

```diff
@@ -0,0 +1,251 @@
+# OpenHW Project Proposal
+# CORE-V IDE
+
+
+
+## Summary of project
+This project proposal is for the CORE-V IDE project. CORE-V IDE will develop an open-source CORE-V version of the Eclipse CDT (C/C++ Development Tools) based IDE (Integrated Developmnent Environmnent). CORE-V IDE will provide the baseline capabilities for a C/C++ IDE suitable for both 32 and 64 bit CORE-V devices. Specific CORE-V IDEs will be able to be instantiated from the CORE-V IDE. 
+

```

Comment:
I'd like to make it explicit right up here that this is meant to support the commercial ecosystem. Something like:

----

In line with the Software TG primary goal of developing a thriving commercial ecosystem, this implementation is intended to be the basis on which others will develop commercial products, both open source and proprietary.

----